### PR TITLE
Add instant message information support for Android

### PR DESCRIPTION
**Pull request by m-k**
I added the usage of the instant message address fields for getContacts().

Also added a fix for Instant Messengers TYPE field.

### CODE REVIEW

**Comment by morenoh149** [comment]
@m-k does ios support im? or does a corresponding ios change need to happen here?

**Comment by m-k** [comment]
@morenoh149 This is the reason why I was forced to implement this PR. The iOS version [already has](https://github.com/morenoh149/react-native-contacts/blob/master/ios/RCTContacts/RCTContacts.m#L468) these attributes in the returned results. I added this on Android to make the return values more consistent. 

Also, these fields are supported in contact form for both platforms.

File: android/src/main/java/com/rt2zz/reactnativecontacts/ContactsManager.java [hunk 1]
```diff
@@ -460,7 +460,7 @@ public void openContactForm(ReadableMap contact, Callback callback) {
             imAddress.put(ContactsContract.Data.MIMETYPE, CommonDataKinds.Im.CONTENT_ITEM_TYPE);
             imAddress.put(CommonDataKinds.Im.DATA, imAccounts[i]);
             imAddress.put(CommonDataKinds.Im.TYPE, CommonDataKinds.Im.TYPE_HOME);
-            imAddress.put(CommonDataKinds.Im.PROTOCOL, CommonDataKinds.Im.CUSTOM_PROTOCOL);
+            imAddress.put(CommonDataKinds.Im.PROTOCOL, CommonDataKinds.Im.PROTOCOL_CUSTOM);
```
Comment (m-k) [inline android/src/main/java/com/rt2zz/reactnativecontacts/ContactsManager.java:463]: [CUSTOM_PROTOCOL](https://github.com/AndroidSDKSources/android-sdk-sources-for-api-level-28/blob/master/android/provider/ContactsContract.java#L6708) - field that contains value of InstantMessaging service name
[PROTOCOL_CUSTOM](https://github.com/AndroidSDKSources/android-sdk-sources-for-api-level-28/blob/master/android/provider/ContactsContract.java#L6713) - data type to use custom service name

**Comment by morenoh149** [comment]
lgtm. Much appreciated!

**Pull request merged**